app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,217 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ *= require foundation_and_overrides
+
+ */
+$red: #ff0000;
+$white: #ffffff;
+$hover: #FFFF99;
+$grey: #333333;
+$foot: #888888;
+
+.container {
+  .text-field {
+    padding: 5px;
+    height: 25px;
+    margin-bottom: 1px;
+  }
+
+  .error-explanation {
+    color: $red;
+    font-size: 10px;
+    font-style: italic;
+    padding-bottom: 15px;
+  }
+
+  .submit-form {
+    background: rgba(0, 0, 0, 0.6);
+    color: $white;
+    font: bold 12px/24px Helvetica, Sans-Serif;
+    margin-bottom: 40px;
+    margin-top: 10px;
+    padding: 1px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
+
+h4 {
+  background: rgba(65, 92, 108, 0.5);
+  color: $white;
+  font: bold 24px/45px Helvetica, Sans-Serif;
+  font-variant: small-caps;
+  letter-spacing: 6px;
+  padding: 10px;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 4px;
+}
+
+.profile-photo {
+  margin-right: 5px;
+  width: 50px;
+}
+
+$facebook-color: #3b5998;
+$twitter-color: #55acee;
+$google-color: #d50f25;
+$text-transform: uppercase;
+$icon-size: 2rem;
+
+.split.button {
+  width: 10%;
+
+  &.facebook {
+    background: $facebook-color;
+    text-transform: $text-transform;
+
+    span {
+      background: darken($facebook-color, 10%);
+
+      &:after {
+        border: none;
+        font-family: "foundation-icons";
+        content: "\f1c4";
+        font-size: $icon-size;
+        line-height: 0.25rem;
+        margin-left: -0.7rem;
+      }
+    }
+  }
+
+  &.twitter {
+    background: $twitter-color;
+    text-transform: $text-transform;
+
+    span {
+      background: darken($twitter-color, 10%);
+
+      &:after {
+        border: none;
+        font-family: "foundation-icons";
```

Comment:
Properties should be ordered border, content, font-family, font-size, line-height, margin-left